react/helpers/getHelpersTypesAccepted.spec.js

```diff
@@ -0,0 +1,76 @@
+import { getHelpersTypesAccepted } from './getHelpersTypesAccepted'
+
+const makeMockFile = ({
+  extension = '.pdf',
+  mime = 'application/pdf'
+} = {}) => ({
+  _id: '123',
+  type: 'file',
+  mime: mime,
+  name: `mockFile${extension}`
+})
+const mockFolder = {
+  _id: '789',
+  type: 'directory',
+  name: 'mockDir'
+}
+
+describe('getHelpersTypesAccepted', () => {
+  it('should have 2 properties', () => {
+    const typesAccepted = getHelpersTypesAccepted()
+
+    expect(typesAccepted).toHaveProperty('isValidFile')
+    expect(typesAccepted).toHaveProperty('isValidFolder')
+  })
+
+  it('isValidFile return "true" by default, if no "itemType" is passed', () => {
+    const { isValidFile } = getHelpersTypesAccepted()
+    const mockFile = makeMockFile()
+
+    expect(isValidFile(mockFile)).toBe(true)
+  })
+
+  it('isValidFile return "true" if "itemType" is "file"', () => {
+    const { isValidFile } = getHelpersTypesAccepted('file')
+    const mockFile = makeMockFile()
+
+    expect(isValidFile(mockFile)).toBe(true)
+  })
+
+  it('isValidFile return "true" if "itemType" has a valid extensions', () => {
```

Comment:
actually I want to rephrase each test to be less close to implementation, more close to the intention. Let's call if you want help for that point 🙏